ui/raidboss/data/triggers/o5n.js

```diff
@@ -51,5 +81,127 @@
       infoText: 'Wind',
       tts: 'wind',
     },
-  ]
+  ],
+  timelineReplace: [
```

Comment:
Ok ok, this is why your patch has so many merge conflicts listed.  I went ahead and did that in upstream cactbot.  If you look at your fork, it says "This branch is 19 commits ahead, 7 commits behind quisquous:master."

One is that I didn't take your commits directly, I squashed them into a single commit.  I also added a few commits of my own past yours.  So, now our two repositories have diverged.

It looks like github has a resolve conflicts button down below for me.  Is that something you can see and sort through?  If you haven't handled merge conflicts before, all the stuff between <<< and === is from your patch, and everything between === and >>> is stuff that's been checked in master.  I went through the timelines and removed things that weren't necessary and also added every missing translation that was needed to fully translate the timeline.  If you just copied and pasted from the savage versions, I suspect you could just always accept my version and be ok.

In the future, I'd recommend uploading patches on separate branches other than master.  (https://learngitbranching.js.org/ is pretty good if you want a tutorial).  That way I can merge things in to upstream and you can re pull back to master and then branch from there to make the next patch.  Usually what most folks do is that they keep master up to date by pulling the latest version from upstream and then do feature work in separate branches that they then delete after it's been merged in.

Reply:
Oh, just noticed this now. I have looked over the conflicts and corrected the issues that I found :)